addons/core/translations/form/en-us.yaml

```diff
@@ -115,3 +115,12 @@ value:
 callback_url:
   label: Callback URL
   help: The URL to which the third-party provider sends the authentication response.  Paste this into the provider configuration.
+vault_path:
+  label: Vault Path
+  help: The path in Vault to request credentials from.
+http_method:
+  label: HTTP Method
+  help: The HTTP method the library uses when requesting credentials from Vault.
+http_request_body:
+  label: HTTP Request Body
+  help: The body of the HTTP request the library sends to Vault when requesting credentials. Only allowed when http_method is set to POST.
```

Comment:
The field name `http_request_body` is probably too generic to mention a specific resource type or cred store type, or particular validation constraints.  I'm not sure how to handle this situation.  Thoughts?

Reply:
Should this be nested to indicate it's vault related?
```
vault:
  path:
  http_method:
  http_body:
```
or for additional structure
```
vault:
  path:
  http:
    method:
    body:
    post_body:
```

Reply:
I'm sort of thinking the pattern here would be to move it under the resource-specific translations into say a `form` key?

Reply:
I like the sound of it. We have tried to keep resource-specific translations pretty flat so how does this look:
```
credential-library:
  title:
  form:
    vault:
      path:
        label:
        description:
      method:
      body:
```

Main reason behind using `vault` tag is coz `method` and `body` descriptions are vault specific.

Reply:
Moved labels into `resources.credential-library.form`